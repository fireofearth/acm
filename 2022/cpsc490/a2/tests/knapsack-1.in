5 4
3 10
4 20
1 15
2 10
1 5
