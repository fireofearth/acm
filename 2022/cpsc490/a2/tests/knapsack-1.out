30
1 15
2 10
1 5
